lib/components/molecules/RenderTable/RenderTable.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+/**
+ *
+ * RenderTable
+ *
+ */
+import React, { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+
+import styles from './RenderTable.style';
+import type { renderTableProps } from './types';
+
+const RenderTable = ({ children, data, className, ...props }: renderTableProps) => {
+  return (
+    <table className={className} {...props}>
+      <thead>
+        <tr>
```

Comment:
Going by this design the table will always be Horizontal one where T-Head is at the top; What if we need T-Head as first cell of each row?